documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_5_Publishing_and_using_Data_Apps.md

```diff
@@ -1,8 +1,23 @@
-## PUBLISHING AND USING DATA APPS
-Data Apps can be used by Connectors for specific data processing or data transformation tasks. They can perform tasks of different complexity, ranging from simple data transformation to complex data analytics. An example of data transformation may be a Data App parsing a single string field with address information and producing a data structure consisting of street name and number, zip code, name of the city, and name of the country.
+## Publishing and using Data Apps
 
-On a conceptual level, Data Apps can be treated the same way as data offerings in the International Data Spaces. Therefore, just as data is provided by a Data Provider using a Connector and registering this Connector at a Broker, Data Apps are created by an App Provider and registered at an App Store (using the App Provider's Connector as a means to communicate with the App Store). As a consequence, App Providers also need to undergo the Onboarding process. However, instead of registering their Connector at a Broker, App Providers  register their Data Apps at an App Store.
+Data Apps can be used by IDS Connectors for specific data processing or transformation tasks. They can perform tasks of different complexity, ranging from simple data transformation to complex data analytics. An example of data transformation may be a Data App parsing a single string field with address information and producing a data structure consisting of street name and number, zip code, name of the city, and name of the country.
 
-In order to be published, certain Data Apps require certification from the Certification Body (see section 3.5.1) (see first step of the process shown in Figure 3.12). When it comes to using a Data App that is offered by an App Store, App Users (Data Provider or Data Consumer) need to execute a process that is very similar to the 'Exchange Data' process described above.
+On a conceptual level, Data Apps can be treated the same way as Data Offerings in the International Data Spaces (see Section [3.3.2](3_3_2_Data_Offering.md)). Therefore, just as a Data Offering is provided by a Data Provider using an IDS Connector registered at an IDS Metadata Broker, Data Apps are created by an App Provider and registered at an IDS App Store using the App Provider's IDS Connector as a means to communicate with the IDS App Store. As a consequence, App Providers also need to follow the Onboarding Process explained in Section [3.3.1](3_3_1_Onboarding.md). Instead of registering their IDS Connector at an IDS Metadata Broker, App Providers register their Data Apps at an IDS App Store.
```

Comment:
A very important deviation from the Metadata Broker is that it stores ONLY the metadata. In the AppStore, metadata AND the apps are stored. This must be made clear here, otherwise there will be misunderstandings. An alternative would be to omit the analogy to the Metadata Broker.

Reply:
addressed in commit [00996cd](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/pull/145/commits/00996cd974b30dd365434dd7bf039a8a9e1257e8)